Scribe/SettingsTab/SettingsViewController.swift

```diff
@@ -1,10 +1,23 @@
 //
-//  SettingsViewController.swift
+//  Copyright (C) 2023 Scribe
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 //
 
 import UIKit
 
-class SettingsViewController: UIViewController {
+final class SettingsViewController: UIViewController {
```

Comment:
I've not inherited the BaseTableViewController for now as the settings screen have another state with a button to add a keyboard, we can tackle this later on to remove those last code duplications.